db/migrate/20230406155827_create_order_books.rb

```diff
@@ -0,0 +1,17 @@
+class CreateOrderBooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :order_books do |t|
+      t.string :order_id, null: false
+      t.references :book, null: false, foreign_key: true
+      t.string :book_title
```

Comment:
Why do we need this column?

Reply:
You mean the column `book_title`. I think the admin sometimes changes the title of the book, so If we don't save the book's title at the time created order, it will be changed and not correct with the title that the customer ordered.

Reply:
If title of the book requires change from admin, shouldn't customer want that change too? Example: a book has wrong title compared to real one, and admin changes it, the customer should see that the book in the order has correct title, rather than the wrong one.

Reply:
I agree with you, I will remove this column.